tasklist/hi.json

```
@@ -0,0 +1,520 @@
+    " CAM_WIDGET_COPY " : " {{value}} " कॉपी करने के लिए क्लिक करें " ,
+    " CAM_WIDGET_COPY_LINK " : " वर्तमान स्थान पर प्रतिलिपि लिंक करें " ,
+    " CAM_WIDGET_CMMN_VIEWER_ERROR " : " चित्र प्रस्तुत करना नहीं कर सका: " ,
+    " CAM_WIDGET_CMMN_VIEWER_LOADING " : " लोड हो रहा है आरेख " ,
+    " CAM_WIDGET_DMN_VIEWER_ERROR " : " तालिका प्रस्तुत नहीं कर सका: " ,
+    " CAM_WIDGET_DMN_VIEWER_LOADING " : " लोडिंग टेबल " ,
+    " CAM_WIDGET_FOOTER_TIMEZONE " : " दिनांक और समय स्थानीय समय क्षेत्र में प्रदर्शित: " ,
+    " CAM_WIDGET_FOOTER_POWERED_BY " : " इसके द्वारा संचालित " ,
+    " CAM_WIDGET_HEADER_TOGGLE_NAVIGATION " : " टॉगल नेविगेशन " ,
+    " CAM_WIDGET_HEADER_MY_PROFILE " : " मेरी प्रोफ़ाइल " ,
+    " CAM_WIDGET_HEADER_SIGN_OUT " : " लॉग आउट करें " ,
+    " CAM_WIDGET_HEADER_SMALL_SCREEN_WARNING " : " । इस आवेदन बड़ी स्क्रीन के लिए कल्पना की है " ,
+    " CAM_WIDGET_SEARCH_MATCH_TYPE " : " के मानदंडों को पूरा कर रहे हैं। " ,
+    " CAM_WIDGET_SEARCH_MATCH_TYPE_ANY " : " कोई भी " ,
+    " CAM_WIDGET_SEARCH_MATCH_TYPE_ALL " : " सभी " ,
+    " CAM_WIDGET_SEARCH_TOTAL_NUMBER_RESULTS " : " कुल परिणामों की संख्या , "
+    " CAM_WIDGET_SEARCH_VARIABLE_CASE_VALUE " : " मान , "
+    " CAM_WIDGET_SEARCH_VARIABLE_CASE_NAME " : " नाम " ,
+    " CAM_WIDGET_SEARCH_VARIABLE_CASE_ATTRIBUTE " : " चर के लिए, के मामले पर ध्यान न दें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TOOLTIP_SET_NULL " : " करने के लिए सेट मूल्य \" अशक्त  " " ,
+    " CAM_WIDGET_VARIABLES_TABLE_PLACEHOLDER_VALUE " : " चर का मान " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TOOLTIP_DEFAULT_VALUE " : " पिछले या डिफ़ॉल्ट मान पर पुन: सेट " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TOOLTIP_UPLOAD " : " एक नई फ़ाइल अपलोड करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_NULL " : " <null> " ,
+    " CAM_WIDGET_VARIABLES_TABLE_EDIT_VARIABLE " : " चर को संपादित करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE_VARIABLE " : " चर हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_SAVE_VARIABLE " : " चर सहेजें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_REVERT_VARIABLE " : " चर को वापस लाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_NAME " : " नाम " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TYPE " : " टाइप " ,
+    " CAM_WIDGET_VARIABLES_TABLE_VALUE " : " मान " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE " : " हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_ABORT " : " बीच में बंद करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DIALOGUE " : " यकीन है कि आप '{{नाम}}' को हटाना चाहते हैं कर रहे हैं " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_INSPECT " : " {{name}} चर " का निरीक्षण करें ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_OBJECT_TYPE " : " ऑब्जेक्ट प्रकार का नाम: " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_SERIALIZATION " : " सीरियल डेटा स्वरूप: " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_SERIALIZED_VALUE " : " धारावाहिक मूल्य " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_BTN_CLOSE " : " बंद करें , "
+    " CAM_WIDGET_VARIABLE_DIALOG_BTN_CHANGE " : " बदलें " ,
+    " CAM_WIDGET_VARIABLE_NULL " : " <null> " ,
```

Please trim the labels
```suggestion
    "CAM_WIDGET_VARIABLE_NULL" : "<null>" ,
```